cmd/client/cmd/service.go

```diff
@@ -0,0 +1,117 @@
+package cmd
+
+import (
+	"context"
+	"log"
+	"os"
+	"time"
+
+	"github.com/jedib0t/go-pretty/v6/table"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	translatev1 "go.expect.digital/translate/pkg/pb/translate/v1"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+const establishConnTimeout = 5 * time.Second
+
+func init() {
+	flags := lsCmd.Flags()
+
+	flags.StringP("address", "a", "localhost:8080", `"translate" service address as "host:port"`)
+	flags.BoolP("insecure", "i", false, `disable transport security (default false)`)
+	flags.DurationP("timeout", "t", establishConnTimeout, `timeout for establishing connection with "translate" service`)
+
+	if err := viper.BindPFlag("address", flags.Lookup("address")); err != nil {
+		log.Panicf("bind address flag: %v", err)
+	}
+
+	if err := viper.BindPFlag("insecure", flags.Lookup("insecure")); err != nil {
+		log.Panicf("bind insecure flag: %v", err)
+	}
+
+	if err := viper.BindPFlag("timeout", flags.Lookup("timeout")); err != nil {
+		log.Panicf("bind timeout flag: %v", err)
+	}
+
+	serviceCmd.AddCommand(lsCmd)
+}
+
+// serviceCmd represents the service command.
+var serviceCmd = &cobra.Command{
+	Use:   "service",
+	Short: "Service holds list of commands for interacting with services.",
+	Run: func(cmd *cobra.Command, args []string) {
+		if err := cmd.Help(); err != nil {
+			log.Panicf("display help: %v", err)
+		}
+	},
+}
+
+// lsCmd represents the lsCmd command.
+var lsCmd = &cobra.Command{
+	Use:   "ls",
+	Short: "Lists services.",
```

Comment:
Let's avoid `.` in the short descriptions.

```suggestion
	Short: "Lists services",
```